lib/assessment.js

```diff
@@ -13,6 +14,9 @@ var min = function min() {
 // and returns the sum of that set
 // assume that all the argument are numbers
 var sum = function sum() {
+  var one = number(one);
+  var two = number(two);
+  return number(one+two);
 
 };
```

Comment:
You are not provided with a function called `number`. You were expected to loop over the `arguments` pseudo-array within the function to get the values passed in and add them up.